spkrepo/tests/common.py

```diff
@@ -397,7 +397,8 @@ def logged_user(self, *args, **kwargs):
         :return: the logged user
         """
         user = self.create_user(*args, **kwargs)
-        self.login(user.email, user.password)
+        response = self.login(user.email, user.password)
+        breakpoint()
```

Comment:
is this breakpoint by intention or should it be removed before merging?

_I am not used in python, but it looks like a leftover from debugging..._

Reply:
Was removed...